documentation/src/object-initializer.md

```diff
@@ -0,0 +1,32 @@
+# Initializer
+
+## Index Initializers
+
+Use index initializers with dictionary.
+This elegantly prevents uninitialized dictionaries.
+
+[Object and Collection Initializers (C# Programming Guide)]: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers#object-initializers-with-collection-read-only-property-initialization
```

Comment:
This link is declared but never used in the text

Reply:
That one I did not see in the example files.
c6018200d4232e68cafe17c654ec66de6b7d2a68